docs/setup_docs.sh

```diff
@@ -36,24 +36,34 @@ function integrate_connector_docs {
 }
 
 
+SKIP_INTEGRATE_CONNECTOR_DOCS=false
+for arg in "$@"; do
+  if [ "$arg" == "--skip-integrate-connector-docs" ]; then
+    SKIP_INTEGRATE_CONNECTOR_DOCS=true
+    break
+  fi
+done
+
 # Integrate the connector documentation
+if [ "$SKIP_INTEGRATE_CONNECTOR_DOCS" = false ]; then
+  echo "Integrate connector docs"
+  rm -rf themes/connectors/*
+  rm -rf tmp
+  mkdir tmp
+  cd tmp
+
+  integrate_connector_docs elasticsearch v3.0
+  integrate_connector_docs aws v4.2
+  integrate_connector_docs cassandra v3.1
+  integrate_connector_docs pulsar v4.0
+  integrate_connector_docs jdbc v3.1
+  integrate_connector_docs rabbitmq v3.0
+  integrate_connector_docs gcp-pubsub v3.0
+  integrate_connector_docs mongodb v1.0
+  integrate_connector_docs opensearch v1.1
+  integrate_connector_docs kafka v3.0
+  integrate_connector_docs hbase v3.0
```

Comment:
Have you checked if the Hugo build fails if nothing has been cloned yet from the external connectors, but the `skip_integrate_connector_docs` is being used? Does the Hugo build fail at that point? If not, then we should be OK. If it fails, it might make sense to first check if the repos have already been cloned and else throw a warning?

Reply:
Hi, @MartijnVisser. Sorry for the later answering. I have checked it, it built success.